website/src/components/OpenCollective.tsx

```diff
@@ -26,6 +26,7 @@ const OpenCollective: React.FC<{}> = () => (
           <a
             href="https://opencollective.com/react-native-elements#backers"
             target="_blank"
+            rel="noopener noreferrer"
           >
             <img src="https://opencollective.com/react-native-elements/backers.svg?width=695" />
```

Comment:
how to alt into image?

Reply:
I'm not sure what you're asking. Are you asking if the image on line 30 should have an `alt` attribute?

That's not really in the scope of the change I was making, but I'm happy to add it if there's some +1s on the request. Omitting the `alt` attribute indicates that the image is a key part of the content and there isn't a textual equivalent. I don't have enough context on this site to decide that.